app/views/notes/_coauthored_notes.html.erb

```diff
@@ -47,7 +47,7 @@
               <p><%= raw strip_tags(sanitize(RDiscount.new(node.body_preview).to_html)) if node.body %></p>
             <% end %>
             <p>
-              <% if logged_in_as(['admin','moderator']) %><a class="btn btn-default btn-xs" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %>
+              <% if logged_in_as(['admin','moderator']) %><a class="btn btn-default" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %>
               <a class="btn btn-default" href="<%= node.path %>"><%= t('notes._notes.read_more') %></a>
```

Comment:
I have run the updated code and my guess was right buttons are coming in different lines. You can just add `style="float: right;"` on line 51 and this is done. Thanks.

Reply:
just pushed the changes to the branch. it should automatically update to the pull request right?

Reply:
yes, it's updated but you changed line 51 of another file `_notes.html.erb` and change is needed in `_coauthored_notes.html.erb` only.

Reply:
alright I reset the previous commit and added the changes to <code>_coauthored_notes.html.erb</code>.